src/lib/coerce.js

```diff
@@ -198,6 +209,8 @@ exports.valObjectMeta = {
         requiredOpts: [],
         otherOpts: ['dflt', 'arrayOk'],
         coerceFunction: function(v, propOut, dflt) {
+            if(isTypedArraySpec(v)) v = decodeTypedArraySpec(v);
```

Comment:
Fix for handling base64 encoded arrays.
cc: https://github.com/plotly/plotly.js/pull/5230 and https://github.com/plotly/plotly.py/pull/4470